isisd/isisd.c

```diff
@@ -565,6 +566,11 @@ void isis_area_destroy(struct isis_area *area)
 
 	area_mt_finish(area);
 
```

Comment:
xfree doesn't need check against null before, right?

Reply:
Agree that's not necessary when looking to the maco which call qfree(). But, well, for me, it is better to check that what you attempt to free is not null. Just because if someone change the macro or qfree() function, you could ran into problem.

Reply:
fixed